edb/lib/std/25-numoperators.edgeql

```diff
@@ -1497,6 +1599,13 @@ std::`^` (n: std::float64, p: std::float64) -> std::float64 {
 };
 
 
+CREATE INFIX OPERATOR
+std::`^` (n: std::bigint, p: std::bigint) -> std::bigint {
```

Comment:
Power cannot be guaranteed to return non-fractional result, because of negative powers, so the return type is `decimal`.